.pipelines/azure-pipelines.yml

```diff
@@ -7,84 +7,81 @@
 trigger:
   branches:
     include:
-    - main
+      - main
```

Comment:
Just to make sure those space weren't intended. Might break the yml

Reply:
Pushed a second commit which resolves this issue.